java/src/jmri/jmrix/cmri/serial/assignment/ListFrame.java

```diff
@@ -433,9 +443,22 @@ public Object getValueAt(int r, int c) {
                 }
                 if (sName == null) {
                     return ("");
+                }
+                
+                if (inputSelected) {
+                    jmri.Sensor s = null;
+                    s = jmri.InstanceManager.sensorManagerInstance().getBySystemName(sName);
```

Comment:
No need to change this, just a suggestion, but code like this can be more readable if you combine the lines:
```
jmri.Sensor s = jmri.InstanceManager.sensorManagerInstance().getBySystemName(sName);
```

Reply:
Yes, of course.  A silly oversight on my part.